RetailCoder.VBE/UI/UnitTesting/TestOutcomeImageSourceConverter.cs

```diff
@@ -24,9 +24,9 @@ public class TestOutcomeImageSourceConverter : IValueConverter
 
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            if (value.GetType() != typeof (TestOutcome))
+            if (value?.GetType() != typeof(TestOutcome))
             {
-                throw new ArgumentException("value must be a TestOutcome");
```

Comment:
We threw for a reason... Only return `null` if the value passed in is `null`. (Unless Mat says the reason we threw isn't needed anymore.)

Reply:
They appear to only be throwing when in the XAML design viewer. I agree that if they are there, it's for a reason. https://chat.stackexchange.com/transcript/message/41200243#41200243 was the suggestion Mug offered. If there's a way to keep them without throwing while in the designer that seems the best option.

Reply:
OK, looks good then.